packages/default/scss/badge/_theme.scss

```diff
@@ -2,7 +2,7 @@
 
     // Solid badges
     @each $name, $color in $theme-colors {
```

Comment:
[question] are we going to add `inherit` as a value for the themeColor prop?

Reply:
the badge is inherit by deault